streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java

```diff
@@ -147,16 +146,22 @@ public void testReprocessingFromScratchAfterResetWithIntermediateUserTopic() thr
             60000);
         // receive only first values to make sure intermediate user topic is not consumed completely
```

Comment:
@mjsax is the `testNo` logic correct in that we only produce when `testNo` is 1?